cmd/flarectl/flarectl.go

```diff
@@ -14,6 +14,13 @@ func main() {
 	app.Name = "flarectl"
 	app.Usage = "Cloudflare CLI"
 	app.Version = "2017.10.0"
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "orgid",
```

Comment:
```suggestion
			Name:  "org-id",
```

Perhaps `org-id` is a touch easier to read?

Reply:
I think your right about that :)